src/cdk/menu/menu.md

```diff
@@ -10,68 +10,65 @@ directives apply their associated ARIA roles to their host element.
 | CdkMenuItemCheckbox | menuitemcheckbox |
+| CdkMenuTrigger      | button           |
 
 ### Getting started
 
 Import the `CdkMenuModule` into the `NgModule` in which you want to create menus. You can then apply
 menu directives to build your custom menu. A typical menu consists of the following directives:
 
-- `cdkMenuTriggerFor` - links a trigger button to a menu you intend to open
-- `cdkMenuPanel` - wraps the menu and provides a link between the `cdkMenuTriggerFor` and the
-  `cdkMenu`
-- `cdkMenu` - the actual menu you want to open
-- `cdkMenuItem` - added to each button
+- `cdkMenuTriggerFor` - links a trigger element to an `ng-template` containing the menu the trigger opens
+- `cdkMenu` - creates the actual menu content that the trigger will open
+- `cdkMenuItem` - added to each item in the menu
 
 <!-- example({
-  "example": "cdk-menu-standalone-menu",
-  "file": "cdk-menu-standalone-menu-example.html"
+  "example": "cdk-menu-standalone-trigger",
+  "file": "cdk-menu-standalone-trigger-example.html"
   }) -->
 
 Most menu interactions consist of two parts: a trigger and a menu panel.
 
 #### Triggers
 
-You must add the `cdkMenuItem` and `cdkMenuTriggerFor` directives to triggers like so,
+`cdkMenuTriggerFor` can be added to any button to make it a trigger for the given menu, or any menu
+item to make it a trigger for a submenu. When adding this directive, be sure to pass a reference to
+the template containing the menu it should open. You can toggle the associated menu using a mouse
+or keyboard.
 
-```html
-<button cdkMenuItem [cdkMenuTriggerFor]="menu">Click me!</button>
-```
+<!-- example({"example":"cdk-menu-standalone-trigger",
+              "file":"cdk-menu-standalone-trigger-example.html", 
+              "region":"trigger"}) -->
 
-Adding `cdkMenuItem` gives you keyboard navigation and focus management. Associating a trigger with
-a menu is done through the `cdkMenuTriggerFor` directive and you must provide a template reference
-variable to it. Once both of these directives are set, you can toggle the associated menu
-programmatically, using a mouse or using a keyboard.
+When creating a submenu trigger, add both `cdkMenuItem` and `cdkMenuTriggerFor` like so,
 
-#### Menu panels
+<!-- example({"example":"cdk-menu-menubar",
+              "file":"cdk-menu-menubar-example.html", 
+              "region":"file-trigger"}) -->
 
-You must wrap pop-up menus with an `ng-template` with the `cdkMenuPanel` directive and a reference
-variable which must be of type `cdkMenuPanel`. Further, the `cdkMenu` must also reference the
-`cdkMenuPanel`.
+#### Menu content
 
-```html
-<ng-template cdkMenuPanel #panel="cdkMenuPanel">
-  <div cdkMenu [cdkMenuPanel]="panel">
-    <!-- some content -->
-  </div>
-</ng-template>
-```
+Menu content is created using the `cdkMenu` or `cdkMenuBar` components. Menus can be always present
+on the page (referred to as an inline menu), or shown on trigger activation (referred to as a popup
```

Comment:
I might structure this like
```markdown
There are two types of menu:
* _inline menus_ are always present on the page
* _pop-up menus_ can be toggled to hide or show by the user
```
(also fwiw my preference is for "pop-up" vs. "popup", but both are broadly acceptable)

Reply:
Done, wound up having to restructure a few sections to make things flow after addressing this. May be worth another review pass over this part